omenai-mobile/screens/register/components/TermsAndConditions/TermsAndConditions.tsx

```diff
@@ -0,0 +1,96 @@
+import { StyleSheet, Text, TouchableOpacity, View } from 'react-native'
+import React, { useState } from 'react'
+import FittedBlackButton from '../../../../components/buttons/FittedBlackButton'
+import BackFormButton from '../../../../components/buttons/BackFormButton'
+import { COLORS } from '../../../../config/colors.config';
+import { ACCEPT_Terms } from '../../../../constants/accetTerms.constants';
+import AntDesign from '@expo/vector-icons/AntDesign';
+
+type TermsAndConditionItemProps = {
+    writeUp: string,
+    isSelected: boolean,
+    handleSelect: () => void
+}
+
+export default function TermsAndConditions({handleBack}: {handleBack: () => void}) {
```

Comment:
The handleBack prop can be added to the type declaration in line 9 and solely selected in line 15